### PR TITLE
Please add ELYTE to your official listings.

### PR DESCRIPTION
**Pull request by philicibine**
Many thanks.  Good job on the fork too!

### CODE REVIEW

File: tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+addr: '0x05d379b48a8622ae4018309e116420db4a38a225'
+decimals: 18
+description: New card and payment system with 1 mil total supply.
+links:
+- Email: mailto:info@elyte.biz
+- Facebook: https://www.facebook.com/EthereumLyte
+- Telegram: https://t.me/joinchat/E6tXj0dThqzzjVlWJRuxLw
+- Website: https://www.elyte.biz
+- Whitepaper: https://main.elyte.biz/?page_id=9
+- BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851
```
Comment (freeatnet) [inline tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml]: Looks like you updated capitalization here, but our format is case-sensitive and prefers `Bitcointalk` instead:

```
Checking tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml
  a links entry should be one of a well-known type (WARN: can 'BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851' be better described by one of well-known types: frozenset({'Telegram', 'Reddit', 'Blog', 'Whitepaper', 'Twitter', 'Bitcointalk', 'WeChat', 'Facebook', 'Github', 'Website', 'Email', 'Linkedin', 'CoinMarketCap', 'Slack'})?)
  Complete.
1 targets x 19 tests = 49 runs, of which: 0 failures, 1 warnings, 0 skipped.
----------------------------------------------------------------------------------------
tokens/0x05d379b48a8622ae4018309e116420db4a38a225.yaml:
  1) Non-critical issue: can 'BitcoinTalk: https://bitcointalk.org/index.php?topic=2618851' be better described by one of well-known types: frozenset({'Telegram', 'Reddit', 'Blog', 'Whitepaper', 'Twitter', 'Bitcointalk', 'WeChat', 'Facebook', 'Github', 'Website', 'Email', 'Linkedin', 'CoinMarketCap', 'Slack'})?
        in test_link_key_known: a links entry should be one of a well-known type
----------------------------------------------------------------------------------------
```

**Comment by philicibine** [comment]
Have updated capitalisation, and added an etherscan link to the verified source.  I hope that's better.

**Comment by philicibine** [comment]
Ok so changed the link to github.

**Comment by philicibine** [comment]
contract source can be verified at [https://etherscan.io/address/0x05d379b48a8622ae4018309e116420db4a38a225#code]etherscan(url)

**Comment by philicibine** [comment]
says checks failed but the links were down when it checked for some reason.  I checked them myself and they are up

**Comment by freeatnet** [comment]
@philicibine this looks like intermittent test failures… you can click on the ❌ (or the check mark) to see the results. The last three failures came from tests being unable to open either https://bitcointalk.org/index.php?topic=2618851 or https://main.elyte.biz/?page_id=9 — though both open fine for me.

The file has no other issues, I'll rerun the tests before merging.

Thank you for being so thorough and persistent!

**Comment by freeatnet** [comment]
@philicibine And now the official website doesn't load for me.

**Comment by philicibine** [comment]
@freeatnet we have been getting ddos'd.  Am currently working to change provider.  Domain will stay the same.  Should change today.

**Comment by freeatnet** [comment]
@philicibine Ugh. Ok, I think the listing file is all good, I'll override the tests in this case.

I still need a working website (I get a 404 on `cparticles.js`) and an official page that mentions the contract address being listed.

**Comment by freeatnet** [comment]
@philicibine Hey, I've really relaxed the timeout in tests, you can try updating your branch if you wanna give it another shot.

**Comment by philicibine** [comment]
@freeatnet I have gained admin access to our site and have updated to show a details page with our token contract address.  

I have also deleted the particles.js plugin.  

Hopefully everything is ok now to merge??  Sorry it's been a hassle.

**Pull request merged**